<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DataProcesser</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow{background-color:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DFD7DF,stop:1 #8F8F8F); }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_run">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#btn_run{	 
background-color: rgba(56, 255, 42, 100);
border:2px solid white;border-radius:10px;
}


QPushButton#btn_run:hover{	 
background-color: rgba(56, 255, 42, 200);
border:2px solid white;border-radius:10px;
}
</string>
          </property>
          <property name="text">
           <string>运行</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_stop">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#btn_stop{
	 
background-color: rgba(255, 15, 19, 100);
border:2px solid white;border-radius:10px;
}
 

QPushButton#btn_stop:hover{
	 
background-color: rgba(255, 15, 19, 200);
border:2px solid white;border-radius:10px;
}
 
</string>
          </property>
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_close">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#btn_close{
	 
background-color: rgba(255, 170, 0, 100);
border:2px solid white;border-radius:10px;
}
 
QPushButton#btn_close:hover{
	 
background-color: rgba(255, 170, 0, 200);
border:2px solid white;border-radius:10px;
}
 





</string>
          </property>
          <property name="text">
           <string>安全退出</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTableWidget" name="tbw_display">
        <property name="styleSheet">
         <string notr="true">#tbw_display{
background-color: rgba(255, 255, 255, 0);
border:2px solid rgb(170, 255, 255);border-radius:10px;

}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLog"/>
   <addaction name="separator"/>
   <addaction name="actionServer"/>
   <addaction name="separator"/>
   <addaction name="actionSet"/>
  </widget>
  <action name="actionLog">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/image/filePath.png</normaloff>:/image/filePath.png</iconset>
   </property>
   <property name="text">
    <string>Log路径</string>
   </property>
  </action>
  <action name="actionServer">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/image/server.png</normaloff>:/image/server.png</iconset>
   </property>
   <property name="text">
    <string>Server配置</string>
   </property>
  </action>
  <action name="actionSet">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/image/set.png</normaloff>:/image/set.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
